server/migrations/20190111131022-create-rating.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Ratings', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    articleid: {
```

Comment:
Please ensure naming conventions are in camelCases... `articleId

Reply:
Done thanks.